library/Focus.hs

```diff
@@ -49,21 +51,48 @@ instance MonadTrans (Focus element) where
 data Change a
   = -- | Produce no changes
     Leave
-  | -- | Delete it
-    Remove
   | -- | Set its value to the provided one
     Set a
-  deriving (Functor, Eq, Ord, Show)
+  | -- | Delete it
```

Comment:
If the move `Set` to be the second constructor, the datatype becomes similar to many common types, which allow coercing between them:
```haskell
data Maybe a = Nothing | Just a
data List a = Nil | Cons a (List a)
```